iab/draft-iab-protocol-transitions.xml

```diff
@@ -913,68 +915,133 @@ Suzanne Woolf
-            <section title="Planning for Replacement">
-                <t> HTTP/1.1 provides a mechanism, Upgrade, to transition to an entirely different
-                    protocol.  That same facility was little used other than to enable the use of
-                    WebSockets <xref target="RFC6455"/>.  However, with performance being a
-                    primary motivation for HTTP/2, a new mechanism was needed to avoid spending
-                    an additional round trip on this negotiation.  A new mechanism was added to
-                    TLS to permit the negotiation of the new version of HTTP: Application Layer
-                    Protocol Negotiation (ALPN) <xref target="RFC7301"/>.  Upgrade was used only
-                    for the unsecured variant of the protocol.
-                </t>
-                <t> ALPN was identified as the way in which future protocol versions would be
-                    negotiated.  The mechanism was well-tested during development of the
-                    specification, which proved that new versions could be deployed safely and
-                    easily using ALPN.  Several draft versions of the protocol were successfully
-                    deployed during protocol development, and version negotiation was never shown
-                    to be an issue.
-                </t>
-                <t> Confidence that new versions would be easy to deploy if necessary lead to a
-                    particular design stance that might be considered unusual in light of the
-                    advice in RFC 5218 <xref target="RFC5218"/>, though is completely consistent
-                    with RFC 6709 <xref target="RFC6709"/>: many of
-                    the ways in which the protocol might be extended were removed unless an
-                    immediate need was understood. This decision was made on the basis that it
-                    would be easier to revise the entire protocol than it would be to ensure that
-                    an extension point was correctly specified and implemented such that it would
-                    be available when needed.
-                </t>
-            </section>
+        <section title="HTTP">
+
+          <t>HTTP has been through several transitions as a protocol.</t>
+
+          <t>The first version <xref target="HTTP0.9"/> was extremely simple, with no headers,
+          status codes, or explicit versioning. HTTP/1.0 <xref target="RFC1945"/> introduced these
+          and a number of other concepts; it succeeded mostly because deployment of HTTP was still
+          relatively new, with a small pool of implementers and (comparatively) small set of
+          deployments and users.</t>
+
+          <t>HTTP/1.1 <xref target="RFC2616"/> was an attempt to make the protocol suitable for the
+          massive scale it was being deployed upon, and to introduce some new features. These
+          included:</t>
+
+          <t><list style="symbols">
```

Comment:
I realize that I lost the xml because that happens with markdown.  Edited comment.